presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/properties/LogicalPropertiesImpl.java

```diff
@@ -164,6 +163,19 @@ public boolean isAtMost(long n)
         return maxCardProperty.isAtMost(n);
     }
 
+    public boolean canBeHomogenized(Set<VariableReferenceExpression> expressions, Set<VariableReferenceExpression> targetVariables)
```

Comment:
add javadoc here about what canBeHomogenized means.

Reply:
It is documented in LogicalProperties like other public methods. I've copied it over here for ease of reference